src/containers/User/UserArea.tsx

```diff
@@ -127,14 +129,14 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({
   ]
   // Only include Snapshots menu item in Dev mode
   if (process.env.NODE_ENV === 'development')
-    adminOptions.splice(1, 0, {
+    configOptions.splice(1, 0, {
       key: 'snapshots',
       text: 'Snapshots',
       value: '/admin/snapshots',
     })
 
-  // Add Admin templates to Admin menu
-  adminOptions.push(
+  // Add Config/Admin templates to Config menu (requires Admin permission)
+  configOptions.push(
```

Comment:
Templates can still be added to the "Config" menu if they are linked to the UI-location "Admin", if we want.